portal-impl/src/portal.properties

```diff
@@ -6104,6 +6104,13 @@
     #
     combo.check.timestamp.interval=1000
 
+    #
+    # Set the maximum number of files allowed in the cache.
+    #
+    # Env: LIFERAY_COMBO_PERIOD_MAX_PERIOD_FILES
+    #
+    combo.max.files=100
```

Comment:
How come we went with 100? I would think that might easily get hit 

cc @dsanz